modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -21,21 +22,27 @@ operations on your computer or server.
 
 Include the required plugin `org.mule.tools.maven.mule-maven-plugin` in your POM file. Do not include the plugin `org.apache.maven.plugins.maven-deploy-plugin` in your POM file or settings file, because it is not compatible with the required plugin.
 
+Using Anypoint Exchange's Maven Facade API version 3 to publish assets to Exchange requires the Mule Maven plugin version 3.5.0 or later.
+
 Note: You must have a unique artifact name and organization ID to publish your asset.
 
 == View Your Organization ID
 
 To construct the elements of a Maven POM file, you need to determine your organization ID:
 
-. Log into Anypoint Platform and click Access Management.
-. Click the organization name for the top level or a business group, and view your Organization ID in the Organization Info screen.
-. Copy the organization ID and add it to the examples that follow for how to set up your project's POM file.
+. Log in to Anypoint Platform and click Access Management.
+. Click the organization name for the top level or a business group, and view your organization ID in the organization info screen.
+. Copy the organization ID so you can add it to your project's POM file when you follow the examples.
 
 == Publish an Asset to Exchange Using Maven
 
-You can use the information in this topic to publish Mule 3 and Mule 4 assets including connectors.
+This repository shows examples of publishing each asset type with Maven: https://github.com/mulesoft-labs/exchange-documentation-samples
+
+The steps to publish an asset with Maven are slightly different for each asset type, and different assets use different Maven plugins, so check the examples in the repository for an example that matches your asset type. For example, use `exchange-mule-maven-plugin` to publish a custom asset, and use `mule-maven-plugin` to publish an extension.
 
-. Modify the groupId of the Maven project to be your Organization ID in your project's POM file:
+The following example shows how to publish a Mule 4 asset such as an application, example, or template.
```

Comment:
This will have to change when we go to versionless Mule.... Or maybe we can start now? We should ask Duke.